backend/prismaCalls.ts

```diff
@@ -65,10 +69,27 @@ export async function uploadMetadataToDatabase(metadata: apiSchema.PackageMetada
             }
         });
     } catch (error) {
-        console.error(`Error in uploadMetadataToDatabase: ${error}`);
+        logger.info(`Error in uploadMetadataToDatabase: ${error}`);
         throw new Error('Failed to upload metadata to the database.');
     }
 }
+
+export async function createPackageHistoryEntry(metadataId: string, userId: number, action: Action): Promise<void> {
+    try {
+        await prisma.packageHistoryEntry.create({
+            data: {
+                metadataId: metadataId,
```

Comment:
The prisma model might have to be changed but I don't think you should have to manually put in a primary key ID. Otherwise, you'll have to keep track of that somehow. Also, before you upload it to the db, you should check if it already exists within the database. "409    Package Exists Already" from the v1spec.yaml. You can just have this in another function and call it before you call this one.